### ISSUE TITLE
FEATURE: Camera button in gallery

### ISSUE THREAD

**Issue by gru2007** [Swiftgram/Telegram-iOS#67]
For now if you hide camera in gallery view you are losing option to take a photo from app. Adding option to open camera from “three dots menu” will bring that function back.

**Comment by deargosep** [comment]
Yes! Woould be nice to also have an option to only disable the preview on the original camera button, as in Turrit

**Comment by Kylmakalle** [comment]
Done in 11.6

**Comment by deargosep** [comment]
When creating a sticker, the preview is still there regardless of camera settings
